lms/djangoapps/discussion_api/api.py

```diff
@@ -39,6 +41,33 @@ def _get_course_or_404(course_key, user):
     return course
 
 
+def _get_thread_and_context(thread_id, request, parent_id=None, retrieve_kwargs=None):
```

Comment:
Should "request" be first for consistency? I noticed in the other functions in api.py, "request" is usually first.